File: JEE/PROJETS/formation-jee/src/main/java/fr/formation/model/Produit.java
package fr.formation.model;

import java.math.BigDecimal;

import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import jakarta.persistence.Table;
import lombok.Getter;
import lombok.Setter;

@Entity // Obligation
@Table(name = "prod")
@Getter @Setter
public class Produit {
    @Id // Obligatoire sur chaque entité
    @GeneratedValue(strategy = GenerationType.IDENTITY) // Auto incrément
    @Column(name = "prod_id")
    private int id;

    private String nom;

    private BigDecimal prix;
}

File: MICROSERVICES/SERVICES/DOTNET/commentaire-service/Models/Produit.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace commentaire_service.Models;


[Table("produit")]
public class Produit
{
    [Key]
    [Column("id")]
    public string Id { get; set; }

    [Column("nom")]
    public string Nom { get; set; } = "";

    [Column("notable")]
    public bool Notable { get; set; }
}
